lib/ui/painting/image_decoder_unittests.cc

```diff
@@ -571,12 +571,77 @@ TEST(ImageDecoderTest, VerifySimpleDecodingWithUpscaling) {
   ASSERT_EQ(SkISize::Make(600, 200), image->dimensions());
 
   ASSERT_EQ(
-      ImageFromCompressedData(data, 900, 300, ImageUpscalingMode::kAllowed,
+      ImageFromCompressedData(data, 900, 300, ImageUpscalingMode::kAllowed, {},
                               fml::tracing::TraceFlow(""))
           ->dimensions(),
       SkISize::Make(900, 300));
 }
 
+TEST(ImageDecoderTest, VerifySimpleDecodingAspectRatioNone) {
```

Comment:
I don't believe we have used value parameterized tests in the engine before (I just found that gtest could do this recently) but this seems like a good place to use them. This pattern is pretty readable too. So, don't a strong opinion either way.